### PR TITLE
Release cadence RFC

### PR DESCRIPTION
**Pull request by coderanger**
Split out the release cadence bits from #227 and add a bit more specific bits. Short and sweet.

### CODE REVIEW

**Comment by shortdudey123** [comment]
Awesome!  If accepted, the major bump would start w/ Chef 13.0 in April 2017 correct?

**Comment by thommay** [comment]
@charlesjohnson FYI

File: new/release-cadence.md [hunk 1]
```diff
@@ -8,14 +8,34 @@ Type: Informational
...
-Chef is released once per month, in the first week of the month unless delayed
-by the Chef Software Release Engineering team.
+Chef follows [Semantic Versioning](https://semver.org/) for releases. Major
+versions (eg. 11.x -> 12.x) will include backwards-incompatible changes, minor
+versions (eg 12.1 -> 12.2) will include new features and bug fixes but will be
+backwards-compatible to the best of our ability. Patch versions are governed
+by [RFC 47](rfc047-release-process.md).
+
+Chef feature releases are promoted to the stable channel once per month, in the
+second week of the month unless delayed by the Chef Software Release Engineering
```
Comment (thommay) [inline new/release-cadence.md]: not sure we need to specify the chef internal structure; maybe just say that we'll aim to do a release a month unless the universe prevents us?

Comment (coderanger) [inline new/release-cadence.md]: Also happy for a better way to say "Nth week unless shit happens" that sounds reasonable, don't want to throw releng under the bus, something that balances standard expectations with reality happening would be good.

Comment (thommay) [inline new/release-cadence.md]: Ah, I meant that I don't think we need to talk about RelEng. Also I would like to eventually get at least the concept of any maintainer doing releases…

```
Chef feature releases are promoted to the stable channel once per month. It is expected that this occur during the second week of the month unless circumstances intervene.
```
Something like that?

Comment (coderanger) [inline new/release-cadence.md]: Much better, thanky!

Comment (charlesjohnson) [inline new/release-cadence.md]: "Chef targets feature releases for the second week of every month" gets it across, but ^^^ is fine.

**Comment by charlesjohnson** [comment]
@shortdudey123 That's correct.

File: new/release-cadence.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Chef feature releases are promoted to the stable channel once per month. It is
+expected that this occur during the second week of the month unless
+circumstances intervene. Additional patch releases for a given feature release
+may be promoted if critical issues are found.
+
+ChefDK is released once per month. It is expected that this occur during the
+fourth week of the month unless circumstances intervene.
+
+Both Chef and ChefDK will prepare a release candidate in the week prior to the
```
Comment (charlesjohnson) [inline new/release-cadence.md]: This is already covered in RFC-047. Can we delete the language here so it's not covered in two places?

**Comment by thommay** [comment]
The language about when an RFC is released shall be moved here from RFC 47. Once that's done, this RFC is accepted.

**Pull request merged**